server/src/internalClusterTest/java/org/opensearch/cluster/coordination/UnsafeBootstrapAndDetachCommandIT.java

```diff
@@ -235,20 +261,24 @@ public void test3MasterNodes2Failed() throws Exception {
 
         logger.info("--> start 1st master-eligible node");
         masterNodes.add(internalCluster().startMasterOnlyNode(Settings.builder()
-                .put(DiscoverySettings.INITIAL_STATE_TIMEOUT_SETTING.getKey(), "0s")
-                .build())); // node ordinal 0
+            .put(DiscoverySettings.INITIAL_STATE_TIMEOUT_SETTING.getKey(), "0s")
```

Comment:
you have updated the indentation at multiple places?

Reply:
IntelliJ automatically updated indentation. Do we have to import any specific rules for IntelliJ for code formatting or defaults are suffice ?

Reply:
@saratvemulapalli can you help with this? Do we have specific code formatter?

Reply:
@saratvemulapalli Thanks, I have checked with that and build is passing successfully without any changes.
So I guess all formatting is according to project rules only. See my [comment](https://github.com/opensearch-project/OpenSearch/pull/599#discussion_r630698243) for build output.